plugins/osdn/ovs/plugin.go

```diff
@@ -34,7 +34,7 @@ func MultiTenantPluginName() string {
 func CreatePlugin(registry *osdn.Registry, multitenant bool, hostname string, selfIP string) (api.OsdnPlugin, api.FilteringEndpointsConfigHandler, error) {
 	plugin := &ovsPlugin{multitenant: multitenant}
 
-	err := plugin.BaseInit(registry, plugin, hostname, selfIP)
+	err := plugin.BaseInit(registry, plugin, multitenant, hostname, selfIP)
 	if err != nil {
 		return nil, nil, err
 	}
```

Comment:
We could log multi-tent plugin info here as BaseInit doesn't need multitenant other than logging.